config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -27,14 +27,13 @@ presubmits:
           command:
             - make
             - test
-          # TODO: direct copy from pull-kubernetes-bazel-test, tune these
           resources:
             limits:
-              cpu: 4
-              memory: "36Gi"
+              cpu: 7.2
+              memory: "43Gi"
             requests:
-              cpu: 4
-              memory: "36Gi"
+              cpu: 7.2
```

Comment:
TODO: iterate on what exactly full machine is. #5085

Reply:
This amount is already in use and should be fine (I looked at current notes with maximum utilization)